playbooks/centurion_teams.yaml

```diff
@@ -0,0 +1,244 @@
+- name: Centurion ERP Teams Setup
+  hosts: |-
```

Comment:
# ToDo - Maintainer

- CI Failure: https://github.com/nofusscomputing/ansible_collection_centurion/actions/runs/10381144478/job/28742085387

``` log

Run actions/add-to-project@244f685bbc3b7adfa8466e08b698b5577571133e
  with:
    project-url: https://github.com/orgs/nofusscomputing/projects/[3](https://github.com/nofusscomputing/ansible_collection_centurion/actions/runs/10381144478/job/28742085387#step:2:3)
Error: Input required and not supplied: github-token

```

Reply:
issue raised, https://github.com/nofusscomputing/action_project/issues/3

- [ ] Add to description

    `- Blocked By: https://github.com/nofusscomputing/action_project/issues/3`

Reply:
- [ ] Add to description

    ``` md
    - Related: https://github.com/nofusscomputing/scratchpad/pull/8#issuecomment-2267537696
    
        _this PR was for the testing of CI on forked Repo's_
    ```